cypress/e2e/CMS/$SiteSections/index.js

```diff
@@ -0,0 +1,95 @@
+/// <reference types="Cypress" />
+import { And, When } from 'cypress-cucumber-preprocessor/steps';
+
+const frontEndBaseUrl = Cypress.env('front_end_base_url');
+const siteSection = Cypress.env('test_site_section');
+
+And('user clicks on {string} sub tab', (contentSubTab) => {
+    cy.get(`ul.admin-list li a:contains("${contentSubTab}")`).click({ force: true });
+});
+
+And('user selects {string} option from Operations for {string}', (option, label) => {
+    cy.get(`.dropbutton >.list > a:contains("${option}")`).last().click({ force: true });
+});
+
+And('user types {string} into Landing Page field', (articleName) => {
+    cy.get(`[class*='form-item-field-landing-page-0-target-id']`).find('input#edit-field-landing-page-0-target-id').type(articleName);
+});
+
+And('user selects {string} from landing page dropdown', (item) => {
+    cy.get(`a:contains('${item}')`).first().should('be.visible').click({ force: true });
+});
+
+And('user expands RELATIONS dropdown', () => {
+    cy.get('details#edit-relations').find('span').first().click({ force: true });
+});
+
+And('user selects {string} from RELATIONS dropdown', (selectSiteSection) => {
+    cy.get('#edit-parent').should('be.visible').select(selectSiteSection);
+});
+
+And('user removes site section', () => {
+    cy.get('#edit-field-site-section-current-items-0-remove-button').click();
+});
+
+When('user selects {string} site section', (selectSiteSection) => {
+    cy.get("input[value='Select Site Section']").click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input[name="computed_path_value"]').type(selectSiteSection);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input#edit-submit-site-section-browser').click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find(`td:contains('${selectSiteSection}')`).first().parent()
+        .find('td.views-field.views-field-entity-browser-select input').check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find("input[id='edit-submit'][value='Select Site Section']").click();
+});
+
+Given('user is navigating to the front end site with path {string}', (purl) => {
+    cy.on('uncaught:exception', (err, runnable) => {
+        if (err.message.includes('Failed to load script https://assets.adobedtm.com')) {
+            return false;
+        }
+        return true;
+    })
+    cy.visit(`${frontEndBaseUrl}${purl}`, { retryOnStatusCodeFailure: true });
+});
+
+Then('the current page is {string}', (title) => {
+    cy.get(`li.usa-sidenav__item a:contains("${title}")`).should('have.attr', 'aria-current', 'page');
+});
+
+And('the following nav children are displayed', (dataTable) => {
```

Comment:
this should be implemented similarly to the same step in side nav tests - so you test the relation to the current page
<img width="790" alt="Screen Shot 2023-01-19 at 12 49 21 PM" src="https://user-images.githubusercontent.com/67657582/213521493-c76dd8a1-1ae5-412c-96e5-dba62cf670e2.png">

Reply:
The assertion passes but getting the following error:
![image](https://user-images.githubusercontent.com/28455973/213583022-e88da8c4-b67d-44e2-bcdf-e8fa1fdda3fd.png)